client/scripts/controllers/chain/ethereum/aave/api.ts

```diff
@@ -41,6 +44,8 @@ export default class AaveApi extends ContractApi<IAaveGovernanceV2> {
     await super.init();
     this._Governance = this.Contract;
 
+    await this._Token.decimals().then((d) => this._decimals = d);
```

Comment:
We have no guarantee that `_Token` is initialized at this point in time.

Reply:
Removed for the time-being, this is used for member's delegation page which we don't have enabled yet for Aave.